backend/models/recipeSchema.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+
+const recipeSchema = new mongoose.Schema({
+    name: String,
+    desciption: String,
+    image: String,
+    ingredients: Array,
```

Comment:
This is my first time seeing Array as a type. 

---I have not fact checked this at all, so I could be totally wrong--- but I would guess that using Array doesn't restrict the type of values that get inputted into the Array. So Ingredients could maybe look like ["salt", 4, True, {name: Cole}]. 

If we wanted to restrict the type of values that gets inserted, using something like [String] does the job. 

Again, ^ the above is just a guess, so no need to change this, just something to keep in mind